### ISSUE TITLE
Feature with 3 OT stages has 2 of them pointing to the same trial ID

### ISSUE THREAD

**Issue by jrobbins** [GoogleChrome/chromium-dashboard#3924]
On feature [Compression dictionary transport with Shared Brotli and Shared Zstandard](https://chromestatus.com/feature/5124977788977152)

There are 3 OT stages.  The first one points to its OT.  The second and third both share the same OT.
* CompressionDictionaryTransport -> https://developer.chrome.com/origintrials/#/view_trial/2583940286203822081
* CompressionDictionaryTransportV2 -> https://developer.chrome.com/origintrials/#/view_trial/3693514644397228033
* CompressionDictionaryTransportV3 -> https://developer.chrome.com/origintrials/#/view_trial/3693514644397228033

I don't see any OT for V3 on OT console or a row in our spreadsheet.

The question is, how did CompressionDictionaryTransportV3's origin_trial_id get set?

**Comment by past** [comment]
Assuming the problem here stems from the same cause as #3910, maybe we don't need any further investigation. @jrobbins can you disconnect the existing trial from the 3rd OT stage so that the owner can request a new trial creation?

**Comment by jrobbins** [comment]
The feature owner reported that the feature entry was back in the initial broken state today.
I investigated further and looked at the code for /cron/associate_origin_trials.

That code seems to correctly look for an existing OT stage that is already associated with the origin trial before trying to associate another stage on the same feature with that origin trial.  So, it should not have reverted back to the broken state.

I looked at the datastore entries and found that the existing OT stage had the origin_trial_id set to the correct value but with type TEXT.  That is not right because TEXT fields are not indexed and will not be found by the cron job when it searches for existing associated OT stages.   It should be type STRING.  I don't see anything in the code that ever used type TEXT.  So, perhaps this was the result of a previous manual datastore edit.

In the pantheon UI, I set the value type to STRING and checked the checkbox to index it.  After that, I ran the cron job twice and verified that it had correctly not associated the 3rd OT stage with the existing OT.  This left the "Request Trial Creation" button available for the feature owner to press.  I responded to the blink-dev thread again to explain and ask the feature owner to use the button.
